lib/mirage.ml

```diff
@@ -729,13 +729,13 @@ let ipv4_qubes db ethernet arp = ipv4_qubes_conf $ db $ ethernet $ arp
 
 let ipv6_conf ?addresses ?netmasks ?gateways () = impl @@ object
     inherit base_configurable
-    method ty = ethernet @-> time @-> mclock @-> ipv6
+    method ty = ethernet @-> random @-> time @-> mclock @-> ipv6
     method name = Name.create "ipv6" ~prefix:"ipv6"
     method module_name = "Ipv6.Make"
     method packages = Key.pure [ package ~sublibs:["ipv6"] "tcpip" ]
```

Comment:
if `opam` would know better about dev packages, we could constrain this here... certainly there should be a `min:"3.0.1"`, but this will make the `ipv6_conf` unusable right now (until a tcpip-3.0.1 is released).

another idea would be to revive `mirage-dev` for this, and use `dev~mirage` as version (or `3.0.1`).  any suggestions?